src/python/pants/backend/python/target_types.py

```diff
@@ -436,7 +444,12 @@ class PythonLibrarySources(PythonSources):
 
 
 class PythonLibrary(Target):
-    """A Python library that may be imported by other targets."""
+    """Python source code.
+
+    A `python_library` does not necessarily correspond to a distribution you publish (see
+    `python_distribution` and `pex_binary` for that). `python_library` is intended for your first-
+    party library code, meaning your code imported by tests and deployables/binaries.
+    """
```

Comment:
I'm not sure how to talk about this. A common source of confusion for new users is thinking that `python_library == python_distribution`.